docs/index.html

```diff
@@ -244,13 +244,15 @@ <h1>Latest Tweets</h1>
 			<h1>Join the Community!</h1>
 			<p>Eclipse SmartHome™ is an open-source project that is driven by
 				its community. Involve yourself now by joining discussions,
-				by reporting issues and by contributing code!</p>
+				by reporting (<a href="https://eclipse.org/security">security related</a>) issues and by contributing code!</p>
```

Comment:
Now "normal" issues are somehow gone. I'd reformulate it to:

> by reporting issues or vulnerabilities and by contributing code! 

(with the link on "vulnerabilities")

Reply:
Why are "normal" issues gone now? I have put "security related" in braces before "issues".

And I would really like to see the term "security" somehow, as I have mentioned that security researchers do search for that.